ClientApp/viking-vault/src/Common/Router.tsx

```diff
@@ -16,12 +16,11 @@ function Router(props: any) {
         <Route path="/register/" exact component={RegisterForm} />
         <Route path="/login" exact component={LoginForm} />
         <Route path="/" exact component={Dashboard}/>
-        <Route path="/automatic-debit" exact component={ViewAutomaticPayments}/>
-        <Route path="/exchange" exact component={ExchangeForm}/>
-        <Route path="/exchange" exact render={() => {return <BlockedServices><ExchangeForm/></BlockedServices>;}}/>
         <Route path="/companies" exact component={Companies}/>
-        <Route path="/notifications" exact component={Notifications}/>
         <Route path="/transfer" exact render={() => {return <BlockedServices><TransferFundsPage/></BlockedServices>;}}/>
+        <Route path="/exchange" exact render={() => {return <BlockedServices><ExchangeForm/></BlockedServices>;}}/>
```

Comment:
`() => 42` is the same as saying `() => { return 42; }`, you can simplify the code here for better readability